mail-filter/pyzor/pyzor-1.0.1_pre20200523.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
```

Comment:
3.9 next time?

Reply:
Back then when the ebuild was created, py3.9 wasn't possible to add, we can improve in the future for sure. First it would be nice to remove the mask and get the updated version to Gentoo :)